package.json

```diff
@@ -25,6 +24,7 @@
     "@angular/platform-browser": "^16.2.0",
     "@angular/platform-browser-dynamic": "^16.2.0",
     "@angular/router": "^16.2.0",
+    "angularx-qrcode": "^16.0.2",
```

Comment:
When running `16.0.2`I get warnings about this package is build as a CommonJS module.

![image](https://github.com/luminator-technology-group/NewsApp/assets/158456476/48480fa5-9237-467f-86c3-374d56e347cc)

This issue is resolved in `17.0.0`

Use the latest version of this package.

Reply:
I run npm install angularx-qrcode@16.0.2  for that but I can use 17